Tests/VisibleItemsProviderTests.swift

```diff
@@ -557,7 +557,9 @@ final class VisibleItemsProviderTests: XCTestCase {
       "Unexpected centermost layout item.")
 
     XCTAssert(details.minimumScrollOffset == nil, "Unexpected minimum scroll offset.")
-    XCTAssert(details.maximumScrollOffset == nil, "Unexpected maximum scroll offset.")
+    XCTAssert(
```

Comment:
This is a correct consequence of the improved boundary-determining code when a user is scrolling very fast.